play27-json-ops/src/main/scala/play/api/libs/json/ops/JsResults.scala

```diff
@@ -0,0 +1,70 @@
+package play.api.libs.json.ops
+
+import play.api.libs.json.{JsError, JsPath, JsResult}
+
+import scala.collection.generic.CanBuildFrom
+import scala.language.higherKinds
+
+/**
+ * Useful helper methods for combining and filtering [[JsResult]]s.
+ */
+object JsResults {
+
+  /**
+   * Filter out all [[JsError]]s and return a collection of results with the most specific type.
+   *
+   * @param results the results to filter
+   * @return a collection of results in whatever order the given collection type maintains
+   */
+  def ignoreErrors[T, C[_]](results: Traversable[JsResult[T]])
```

Comment:
While I think that the `C[_]` argument should be the first type parameter, I didn't want to make this different than what is already available in previous versions.